test/support/fixtures/accounts_fixtures.ex

```diff
@@ -19,7 +19,6 @@ defmodule Bokken.AccountsFixtures do
 
   def valid_guardian_attributes(attrs \\ %{}) do
     Enum.into(attrs, %{
-      photo: Avatar.image_url("#{System.unique_integer()}.png"),
```

Comment:
If you are removing this, you should have a module to separately test it. We can't lose _track_ of it.